ReviewComments/README.md

```diff
@@ -0,0 +1,21 @@
+Pull Request Review Comments ![size 1.6GB](https://img.shields.io/badge/size-1.6GB-green.svg)
+===========
+
+The dataset was extracted from [GH Archive](https://www.gharchive.org/) and consists of:
+
+1. [45.8 million pull request review comments](https://drive.google.com/open?id=1rk6OTDrD09xVU0o_w8_dvtsLaeeUgwmP) extracted from GitHub from 2015 to 2018 included - 1.6 GB (xz-compressed)
```

Comment:
```suggestion
1. [45.8 million pull request review comments](https://drive.google.com/open?id=1rk6OTDrD09xVU0o_w8_dvtsLaeeUgwmP) since 2015 till 2018 - 1.6 GB (xz-compressed)
```

What about months?